bip-schnorr.mediawiki

```diff
@@ -140,6 +140,9 @@ To sign ''m'' for public key ''dG'':
 * Let ''e = int(hash(bytes(x(R)) || bytes(dG) || m)) mod n''.
 * The signature is ''bytes(x(R)) || bytes(k + ed mod n)''.
 
+'''Do not reuse above deterministic derivation of ''R'' in multi-party signature schemes such as MuSig (see the [https://eprint.iacr.org/2018/068 MuSig paper] for details)!'''
+Otherwise the signature will leak the secret key.
```

Comment:
Would tone that down a little bit.
"Otherwise signatures can leak the secret key" (a MuSig signature alone does not necessarily leak the secret key and there may be other multi-signature schemes with entirely different properties.)